extensions/windows-event-log/wel/WindowsEventLog.cpp

```diff
@@ -158,51 +159,59 @@ std::string WindowsEventLogMetadataImpl::getEventData(EVT_FORMAT_MESSAGE_FLAGS f
   return event_data;
 }
 
-std::string WindowsEventLogHandler::getEventMessage(EVT_HANDLE eventHandle) const {
+nonstd::expected<std::string, DWORD> WindowsEventLogHandler::getEventMessage(EVT_HANDLE eventHandle) const {
```

Comment:
I suggest using `std::error_code` as the unexpected type.

Reply:
again thanks, its much better this way 👍 https://github.com/apache/nifi-minifi-cpp/pull/1434/commits/e142a7d9796ac63ccd4758a9e5a570dae1509fc9#diff-72a907e167b376aa07b9e4ee412b95e974d9fa89c80477a735f4c23e966038b3R75